docs/kratos/manage-identities/25_import-user-accounts-identities.mdx

```diff
@@ -453,6 +453,86 @@ $hmac-sha512$OTFmODY0ZTI1NmU0ZjVhYjhiMDViZGFmNGVmNGZmMGVlNTY4ODYwNWJhYTk4MTk2OTg
 </Tabs>
 ````
 
+### Password migration using a web hook
+
+If you want to import users, but do not have access to the hashed password, or the hashed password is in a format that Ory
+Identities does not support, you can use a web hook to migrate the password.
+
+The web hook will be called when the user logs in, and will receive the user's identifier and password. If the web hook returns
+successfully, the user will be logged in and the hashed password will be stored.
+
+The following steps are necessary to set up password migration using a web hook:
+
+#### Step 1: Import an identity with an empty password hash and `use_migration_hook` set to `true`
```

Comment:
For "stepped" guides, we use lists in other places. IMO, this should be consistent across all docs, so I'd prefer a list here, similar to here: https://www.ory.sh/docs/hydra/guides/custom-ui-oauth2#run-the-sample-application
(source: https://github.com/ory/docs/blob/master/docs/hydra/guides/custom-ui-oauth2.mdx#L105-L133)

Reply:
fixed